com.unity.formats.alembic/Runtime/Unity.Formats.Alembic.Runtime.api

```diff
@@ -29,8 +29,8 @@ namespace UnityEngine.Formats.Alembic.Importer
     [UnityEngine.DisallowMultipleComponent] public class AlembicCurves : UnityEngine.MonoBehaviour
     {
         public event UnityEngine.Formats.Alembic.Importer.AlembicCurves.OnUpdateDataHandler OnUpdateData;
-        public int[] CurvePointCount { get; }
         public UnityEngine.Vector3[] Positions { get; }
+        public int[] StrideArray { get; }
```

Comment:
That should be CurveOffsets right?

Reply:
Auto-generated file. forgot to commit it :(
Should really port the unit test I did for recorder that checks for such things...